docs/user/REST.rst

```diff
@@ -2,6 +2,261 @@
 REST Service
 ============
 
-This section will describe how to easily expose your PyTorch
-:class:`~torch.nn.Module` as a REST-API with the help of skorch and
-`palladium <https://github.com/ottogroup/palladium>`_.
+In this section we'll take the RNN sentiment classifer from the
+example `Predicting sentiment on the IMDB dataset
+<https://github.com/dnouri/skorch/blob/master/examples/rnn_classifer/RNN_sentiment_classification.ipynb>`_
+and use it to demonstrate how to easily expose your PyTorch module on
+the web using Skorch and another library called `Palladium
+<https://github.com/ottogroup/palladium>`_.
+
+With Palladium, you define the dataset, the model, and Palladium
+provides the framework to fit, test, and serve your model on the web.
+Palladium comes with its own documentation and a `tutorial
+<http://palladium.readthedocs.io/en/latest/user/tutorial.html>`_,
+which you may want to check out to learn more about what you can do
+with it.
+
+The way to make the dataset and model known to Palladium is through
+its configuration file.  Here's the part of the configuration that
+defines the dataset and model:
+
+.. code:: python
+
+    {
+        'dataset_loader_train': {
+            '__factory__': 'model.DatasetLoader',
+            'path': 'aclImdb/train/',
+        },
+
+        'dataset_loader_test': {
+            '__factory__': 'model.DatasetLoader',
+            'path': 'aclImdb/test/',
+        },
+
+        'model': {
+            '__factory__': 'model.create_pipeline',
+            'use_cuda': True,
+        },
+
+        'model_persister': {
+            '__factory__': 'palladium.persistence.File',
+            'path': 'rnn-model-{version}',
+        },
+    }
+
+You can save this configuration as ``palladium-config.py``.
+    
+The ``dataset_loader_train`` and ``dataset_loader_test`` entries
+define where the data comes from.  They refer to a Python class
+defined inside the ``model`` module.  Let's create a file and call it
+``model.py``, put it in the same directory as the configuration file.
+We'll start off with defining the dataset loader:
+
+.. code:: python
+
+    import os
+    from urllib.request import urlretrieve
+    import tarfile
+
+    from palladium.interfaces import DatasetLoader as IDatasetLoader
```

Comment:
There's a name conflict otherwise because I'm calling the DatasetLoader in this file DatasetLoader. :)

Interfaces in [other languages](https://msdn.microsoft.com/en-us/library/8bc1fexb(v=vs.71).aspx) are often prefixed with I, so that's where the I comes from.

Reply:
okay, that convention was new to me